site/en/blog/hidden-until-found/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Making collapsed content accessible with hidden=until-found'
+description: >
+  How this new attribute value can ensure that content within accordion sections can be found and linked to. 
+layout:
+  'layouts/blog-post.njk'
+date: 2022-04-27
+authors:
+  - jarhar
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/bt6eW4gluWeTMRNfvKa6.jpg'
+alt: >
+  A cat hiding behind a curtain.
+tags:
+  - chrome
+  - html
+---
+
+Collapsing sections of content is a common UI pattern, sometimes described as [an accordion](https://www.smashingmagazine.com/2017/06/designing-perfect-accordion-checklist/), however content hidden in the collapsed sections becomes impossible to search using a find-in-page search. Also, it isn't possible to link to [text fragments](https://web.dev/text-fragments/) inside collapsed regions.
+
+The `hidden=until-found` HTML attribute and `beforematch` event can solve these problems. By adding `hidden=until-found` to the container for your hidden content, you make it possible for the browser to match text in that hidden region, and reveal the section if a match is found.
```

Comment:
Replace "match text in that hidden region" with "search text in that hidden region"?